### ISSUE TITLE
\Phalcon\Annotations\Reader crashs php if annotation does not contain any php doc type

### ISSUE THREAD

**Issue by DasRed** [phalcon/cphalcon#10346]
# Reproduction
This code let crash PHP.

**Example 1**
```php
$reader = new \Phalcon\Annotations\Reader();
class XXX
{

	/**
	 * after the fetching convert data to correct type
	 */
	public function afterFetch()
	{
	}
}

print_r($reader->parse('XXX'));
```

**Example 2**
```php
/**
 * model for database table "bank_account"
 */
class XXX extends \Phalcon\Mvc\Model
{
}

$reader = new \Phalcon\Annotations\Reader();
print_r($reader->parse('XXX'));
```

# Environment
PHP Version: 
PHP 5.6.4 (cli) (built: Dec 17 2014 13:20:35)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend Debugger v8.0.0, Copyright (c) 1999-2014, by Zend Technologies

Win7 : Microsoft Windows [Version 6.1.7601]

Phalcon 2.0.1

# Workaround

**Example 1**
```php
$reader = new \Phalcon\Annotations\Reader();
class XXX
{

	/**
	 * after the fetching convert data to correct type
	 *
	 * @return void
	 */
	public function afterFetch()
	{
	}
}

print_r($reader->parse('XXX'));
```

added: @return void

**Example 2**
```php

/**
 * @desc model for database table "bank_account"
 */
class XXX extends \Phalcon\Mvc\Model
{
}

$reader = new \Phalcon\Annotations\Reader();
print_r($reader->parse('XXX'));
```
added: @desc to PHPDocBlock

**Comment by andresgutierrez** [comment]
This is already fixed in the 2.0.x branch

**Comment by DasRed** [comment]
@andresgutierrez I used the compiled downloadable version 2.0.1. If this not the 2.0.x branch?
If not, so release and deploy a brand new 2.0.x extension for Windows , then i can test it.
I don't want it compile on my own ;)

**Comment by Green-Cat** [comment]
@andresgutierrez Fixed in 2.0.1.
